app/utils/formHelpers.ts

```diff
@@ -75,47 +110,45 @@ const renderYesNoString = (value: unknown): string => {
 };
 
 /**
- * Return the `label` column from a related lookup table
- */
-export const renderLookupLabel = <T extends Record<string, unknown>>(
```

Comment:
here's one of those functions that was hardcoded to deal with with our lookup tables in the `lookups` schema, but not for other types of relationships.